README.md

```diff
@@ -192,6 +192,22 @@ To run black:
 black .
 ```
 
+### Create and run db migration script
```

Comment:
Depending on @isabelcosta response to [this](https://github.com/anitab-org/mentorship-backend/pull/1038#issuecomment-831534613) post, there might be a slight change here (we need to clearly specify for which purpose the contributor is handling the migration script and address each section accordingly).